File: Python/9(Special Pythagorean triplet)_D5.py
import math
a = 1
b = 1
c = 1
while a <= 500:
    while b <= 500:
        while c <= 500:
            total = a + b + c
            cCheck = a**2 + b**2
            if total == 1000 and (c**2 == cCheck):
                print(str(a) + " + " + str(b) + " = " + str(c) +": " + str(total))
                print(a*b*c)
                a = 1000
                b = 1000
                c = 1000
            c += 1
        c = b + 1
        b += 1
    b = a + 1
    a += 1

File: Javascript/9(Special Pythagorean triplet)_D5.js
var solved = false;
var solution;
for(a = 1; a<997 && solved == false; a++){
    for(b = a+1; b<998 && solved == false; b++){
        for(c=b+1; c<999; c++){
            if((a+b+c) == 1000 && (Math.pow(a, 2) + Math.pow(b, 2) == Math.pow(c, 2))){
                solved = true;
                solution = a*b*c;
                break;
            }
        }
    }
}
console.log(solution);